EIPS/eip-831.md

```diff
@@ -1,17 +1,15 @@
 ---
 eip: 831
 title: URI Format for Ethereum
-author: ligi <ligi@ligi.de>
+description: A way of creating Ethereum URIs for various use-cases.
+author: ligi (@ligi)
+discussions-to: https://ethereum-magicians.org/t/eip-831-uri-format-for-ethereum/10105
+status: Draft
 type: Standards Track
 category: ERC
-status: Stagnant
 created: 2018-01-15
```

Comment:
```suggestion
created: 2018-01-15
requires: 67, 681
```
Note: At the moment `requires` means "required reading to fully understand this EIP" and it doesn't necessarily mean that someone implementing this EIP needs to also implement either of these EIPs